session/session.go

```diff
@@ -196,7 +196,7 @@ func (manager *Manager) getSid(r *http.Request) (string, error) {
 			}
 		}
 
-		return sid, nil
+		return sid, errs
```

Comment:
We can not return errs here since the invocation `sid = r.FormValue(manager.config.CookieName)` maybe successful.

Reply:
Success is nil. Failure is err